tower-balance/src/load/peak_ewma.rs

```diff
@@ -120,11 +116,7 @@ where
 
 // ===== impl PeakEwma =====
 
-impl<S, I> PeakEwma<S, I>
-where
-    S: Service,
-    I: Instrument<Handle, S::Response>,
-{
+impl<S, I> PeakEwma<S, I> {
```

Comment:
The bounds on this impl block had to be removed because the `R` type parameter on `Service` would be unbounded without a `PhantomData` field on `PeakEwma`.